### PR TITLE
GrafanaExperimental: update dependency

### PR DESCRIPTION
**Pull request by torkelo**
Just a small update to use latest published package

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -19355,7 +19365,7 @@ __metadata:
     "@grafana/e2e": "workspace:*"
     "@grafana/e2e-selectors": "workspace:*"
     "@grafana/eslint-config": 2.5.1
-    "@grafana/experimental": canary
+    "@grafana/experimental": ^0.0.2-canary.6
```
Comment (mckn) [inline yarn.lock:19368]:
```suggestion
    "@grafana/experimental": latest
```

I guess that should resolve to the same package. Not really sure about the ideas of having canary and latest for this package.

Comment (torkelo) [reply yarn.lock:19368]: @mckn ?? the lock file I usually let yarn handle

Comment (torkelo) [reply yarn.lock:19368]: I ran `yarn up @grafana/experimental`, is there another command to run?

Comment (mckn) [reply yarn.lock]: ah, sorry - I commented on the wrong file 🙈

Comment (mckn) [reply yarn.lock]: or I read this PR totally wrong. I thought you had specified the version in the package.json. But then you can ignore this comment.

**Review by mckn — APPROVED** [review]
LGTM!

**Review by dprokop — APPROVED** [review]

**Pull request merged**